compatibility.bs

```diff
@@ -1055,11 +1057,105 @@ On mobile platforms, including smaller iPad form factors
   </tbody>
 </table>
 
+
+<h2 id="removal">Deprecation and Removal</h2>
+
+<em>This section is non-normative.</em>
+
+While the web platform stives for backwards-compatibility, there are occasions which require the
+outright removal or discouraged use of some features. To support making these changes
+in the least disruptive and most consistent way for developers, we document them here. This is most
+beneficial as a decalration of intent by browser engines to act together in a well reasoned way.
+
+<h3 id="removal-features">Feature List</h3>
+
+<table>
+  <thead>
+    <tr>
+      <th><dfn for="deprecated feature list">Feature</dfn></th>
+      <th><dfn for="deprecated feature list">Deprecated from</dfn></th>
+      <th><dfn for="deprecated feature list">Removal</dfn></th>
+      <th><dfn for="deprecated feature list">Replacement</dfn></th>
+      <th><dfn for="deprecated feature list">Justification</dfn></th>
+      <th><dfn for="deprecated feature list">Automated Tools</dfn></th>
```

Comment:
As noted in some previous comments, it seems like we'll be cramming a lot of information into this horizontal table. I wonder if it would be reasonable to either:

* Simplify the table by dropping the last three columns, and instead requiring an explainer link (that presumably would contain this information). Or,
* Change the table into a more vertical structure. Just bulleted lists with relevant text?

Reply:
I think the first option is a bit better, if only to give information "at a glance". I envisioned justification as a link somewhere anyway, so some suggestion of what content is most important would be a reasonable substitute.